src/SmartComponents/PatchSetDetail/PatchSetDetail.js

```diff
@@ -56,7 +56,7 @@ import { defaultReducers } from '../../store';
 import { inventoryEntitiesReducer, modifyTemplateDetailSystems } from '../../store/Reducers/InventoryEntitiesReducer';
 import { systemsListColumns } from '../Systems/SystemsListAssets';
 import { processDate } from '@redhat-cloud-services/frontend-components-utilities/helpers';
-import { ID_API_ENDPOINTS, useOnSelect } from '../../Utilities/useOnSelect';
+import { ID_API_ENDPOINTS, useOnSelect } from '../../Utilities/hooks';
```

Comment:
here there two import sections coming from the same path. we should combine them